addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2524,6 +2524,11 @@ const SnippetOptionWidget = Widget.extend({
                         values = values.join(' ').replace(color, '').trim();
                         value = `${color} ${values}${inset ? ' inset' : ''}`;
                     }
+                    if (params.cssProperty === 'border-width' && value.endsWith('px')) {
```

Comment:
In fact:

```javascript
const borderWidthCssProps = weUtils.CSS_SHORTHANDS['border-width'];
[...]
if (borderWidthCssProps.includes(cssProp) && value.endsWith('px')) {
```

Otherwise we would theoretically still have the bug for an option targeting a specific border width only no?